File: problemas/iniciante/2029/2029.exs
defmodule Modulo do
  def leitura() do
    try do
      {v, _} = IO.gets("") |> String.trim() |> Float.parse
      {d, _} = IO.gets("") |> String.trim() |> Float.parse
      area = 3.14 * (d / 2.0) * (d / 2.0)
      altura = v / area
      IO.puts :io_lib.format("ALTURA = ~.2f", [altura])
      IO.puts :io_lib.format("AREA = ~.2f", [area])
      leitura()
    rescue
      FunctionClauseError -> []
    end
  end
end

Modulo.leitura()

File: problemas/iniciante/2029/2029.rb
pi = 3.14

while entrada = gets
  v = entrada.to_f
  d = gets.to_f / 2.0

  area = pi * d * d
  altura = v / area

  puts format('ALTURA = %.2f', altura)
  puts format('AREA = %.2f', area)
end
